### DISCUSSION TITLE
Final hosting?

### DISCUSSION THREAD

**Discussion by irisoren-sg** [best-practice-and-impact/aqua_book_revision#63] [General]
@zarbspace @rebecca-wodcke-gad  @Hurstharrier 

Is the book's final location going to be GitHub, or will it be hosted elsewhere?

**Comment by Hurstharrier** [comment]
The intended final location is on the Analysis Function Web site




***@***.***

Alec Waterhouse
Deputy Director: Energy Infrastructure and Markets Analysis
I cover: Systems and Networks, Modelling, Research, Nuclear
Theresa Chambers covers: CCUS, Renewables, Energy Portfolio/strategy
Tom Griffiths covers: Markets (inc. REMA), Affordability, Security
Tel: 0300 068 2929 | 0778 994 7785 | ***@***.***
***@***.***%0b>PA: Zoë Beaverstock| T: | e: ***@***.******@***.***>


From: Iris Oren ***@***.***>
Sent: Friday, June 28, 2024 10:14 AM
To: best-practice-and-impact/aqua_book_revision ***@***.***>
Cc: Waterhouse, Alec (Energy Security) ***@***.***>; Mention ***@***.***>
Subject: [best-practice-and-impact/aqua_book_revision] Final hosting? (Discussion #63)


@zarbspace<https://github.com/zarbspace> @rebecca-wodcke-gad<https://github.com/rebecca-wodcke-gad> @Hurstharrier<https://github.com/Hurstharrier>

Is the book's final location going to be GitHub, or will it be hosted elsewhere?

-

**Comment by Hurstharrier** [comment]
Here is an example

https://analysisfunction.civilservice.gov.uk/policy-store/an-introductory-guide-to-mcda/



***@***.***

Alec Waterhouse
Deputy Director: Energy Infrastructure and Markets Analysis
I cover: Systems and Networks, Modelling, Research, Nuclear
Theresa Chambers covers: CCUS, Renewables, Energy Portfolio/strategy
Tom Griffiths covers: Markets (inc. REMA), Affordability, Security
Tel: 0300 068 2929 | 0778 994 7785 | ***@***.***
***@***.***%0b>PA: Zoë Beaverstock| T: | e: ***@***.******@***.***>


From: Iris Oren ***@***.***>
Sent: Friday, June 28, 2024 10:14 AM
To: best-practice-and-impact/aqua_book_revision ***@***.***>
Cc: Waterhouse, Alec (Energy Security) ***@***.***>; Mention ***@***.***>
Subject: [best-practice-and-impact/aqua_book_revision] Final hosting? (Discussion #63)


@zarbspace<https://github.com/zarbspace> @rebecca-wodcke-gad<https://github.com/rebecca-wodcke-gad> @Hurstharrier<https://github.com/Hurstharrier>

Is the book's final location going to be GitHub, or will it be hosted elsewhere?

-
